### PR TITLE
Fix Helm chart docs

### PR DESCRIPTION
**Pull request by adriangonz**
**What this PR does / why we need it**:
Fix template for Helm chart docs, which was breaking the options table.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **adriangonz**
You can assign the PR to them by writing `/assign @adriangonz` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Tue Jul 28 12:37:22 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2207/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2207 --build=1

**Comment by seldondev** [comment]
Tue Jul 28 12:37:26 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2207/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2207 --build=2

File: helm-charts/README.md.tpl [hunk 1]
```diff
@@ -40,7 +40,7 @@ using:
 helm install $MY_MODEL_NAME seldonio/{{ .Name }} --namespace $MODELS_NAMESPACE
 ```
 
-{{- end -}}
```
Comment (RafalSkolasinski) [inline helm-charts/README.md.tpl:43]: If `if` and `else` have no `-` why did they decided the have a trailing `-` in the `end`...

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[RafalSkolasinski](https://github.com/SeldonIO/seldon-core/pull/2207#pullrequestreview-456599380 "Approved")*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [RafalSkolasinski]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: helm-charts/README.md.tpl (hunk 1 above)
Comment (adriangonz) [reply helm-charts/README.md.tpl:43]: They do magic things. I think that the `-` at the beginning removes the previous new line. And the `-` at the end, removes the following new line (both cases if there is any).

**Pull request merged**